lib/cog_api/resources/bundle_version.ex

```diff
@@ -30,4 +33,14 @@ defimpl Poison.Decoder, for: CogApi.Resources.BundleVersion do
     do: %{value | commands: []}
   defp default(value, _),
     do: value
+
+  defp set_status(%{incompatible: true}=version),
+    do: %{version | status: "Incompatible"}
+  defp set_status(%{enabled: true}=version),
+    do: %{version | status: "Enabled"}
+  defp set_status(%{enabled: false}=version),
+    do: %{version | status: "Disabled"}
+  defp set_status(version),
+    do: %{version | status: nil}
```

Comment:
It can. Sometimes we get just a simple list of bundle versions back. The only thing set is the name and id.